### DISCUSSION TITLE
Space-charge calculations

### DISCUSSION THREAD

**Discussion by JohnMSmith** [MethodicalAcceleratorDesign/MAD-X#974] [Q&A]
I'm interested in space-charge calculations. Unfortunately there's not enough info in user's guide to understand what actions should be taken in order for space-charge effects to be accounted. 
I wish to calculate incoherent tune shift (spread). 
As I understood for start I need to set OPTION, bborbit; and install BEAMBEAM elements in the ring with BBDIR=1. Then I need to create a gaussian distribution. What else am I missing?

**Comment by ldeniau** [comment]
Andrea, could you say a word about the status of the SC module in the last MAD-X release 5.08.00?
